.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -30,7 +30,10 @@ body:
   - type: textarea
     attributes:
       label: Package version(s)
-      description: Which version(s) did you observe the bug?
+      description: |
+        Which version(s) did you observe the bug in? 
+
+        Please provide the exact version(s) instead of using range `^`. If you’re not sure which version your project is using, run `npm ls [package name]` or `yarn why [package name]` to find out.
```

Comment:
```suggestion
        Please provide the exact version(s) instead of a [range](https://github.com/npm/node-semver#advanced-range-syntax) (`^`, `~`, `>`, `*`, `X`). If you’re not sure which version your project is using, run `npm ls [package name]` or `yarn why [package name]` to find out.
```